src/SSVOpenHexagon/Core/HGUpdate.cpp

```diff
@@ -23,6 +23,8 @@ namespace hg
 
 void HexagonGame::update(ssvu::FT mFT)
 {
+    mFT *= Config::getTimescale();
```

Comment:
This slows down or speeds up the entire simulation. It is useful to test high difficulty multipliers "in slowmo".